api/providers/tasks.py

```diff
@@ -453,10 +453,16 @@ def handle_registration_row(row, initiator, provider, schema, auto_approval=Fals
         )
         # Remove all contributors except the initiator if created from an existing node
         if node:
+            # Temporarily make initiator contributor visible so that removal of the others can succeed.
```

Comment:
This is temporary since 15 lines below the visibility is set to `False` no matter what the previous visibility is.